src/AddIns/Uno.UI.MediaPlayer.Skia.Gtk/GtkMediaPlayer.cs

```diff
@@ -200,13 +200,7 @@ public void Stop()
 				this.Log().Debug("Stop");
 			}
 
-			_ = Task.Run(() =>
-			{
-				// Required to be on a background thread
-				// to avoid deadlocks
-				_mediaPlayer.Stop();
-			});
-
```

Comment:
@jeromelaban why was this causing a deadlock? Asynchronously stopping is problematic because we don't want a race condition when reading `_mediaPlayer.State` afterwards.

Reply:
@Youssef1313 will be able to tell.

Reply:
@jeromelaban I'm not really sure why this block of code originally existed.